modules/ROOT/pages/microsoft/microsoft-365-ops-connector.adoc

```diff
@@ -82,49 +92,64 @@ You can use this connector in Anypoint Studio by first downloading it from Excha
-** Authorization URL - Tenant specific OAuth2.0 endpoint. Format: `+https://login.microsoftonline.com/{tenant}/oauth2/authorize+` (follow instructions presented in the above link for specific information on how to obtain the endpoint)
-** Access token URL - Tenant specific endpoint for requesting the access token; format: `+https://login.microsoftonline.com/{tenant}/oauth2/token+` (follow instructions presented in the above link for specific information on how to obtain the endpoint).
-** Listener config - Name of the HTTP listener instance created in studio flow
-** Callback path - The URL of your app, where authentication responses can be sent and received by your app (follow instructions presented in the above link for specific information on how to obtain the endpoint).
-** Authorize path - The URL which triggers the OAuth mechanism
-** External callback URL - Optional: configure this if the callback URL is not accessible directly (example: when using proxies) example format: `+http://localhost:8081/callback+`.
-+
-image::microsoft/microsoft-365-ops-as-oauth2.png[Studio OAuth configuration]
-+
-.. OAuth 2.0 Username-Password
-+
-** Username - Username used to initialize the session.
-** Password - Password used to authenticate the user.
-** Resource - The application ID URI of the web API (secured resource). This must be the root URI, without specifying the version. Example: `+https://YOUR_ORG.operations.dynamics.com/+`
-** Client ID - The application ID assigned to your app when you registered it with Azure Active Directory. You can find this in the Azure Portal (portal.azure.com). Click Active Directory, click the directory, choose the application, and click Configure.
-** Client Secret - The application secret that you created in the app registration portal for your app. This should not be used in a native app, because client secrets cannot be reliably stored on devices. This is required for web apps and web APIs, which have the ability to store the client secret securely on the server side. You can find this in the Azure Portal (portal.azure.com).
-** Token Request Endpoint - The token endpoint that is called to get the access token. Example: `+https://login.windows.net/TENANT_ID/oauth2/token+` where TENANT_ID is the Azure Active Directory ID.
-+
-image::microsoft/microsoft-365-ops-as-oauth-username-pass.png[Studio OAuth Username Password configuration]
+Drag and drop the connector to the Studio Canvas.
+
+Create a global element to:
+
+* <<Configure OAuth 2>>
+* <<Configure OAuth 2.0 Username Password>>
+
+=== Configure OAuth 2
+
+[%header,cols="30s,70a"]
+|===
+|Field |Description
```

Comment:
ditto re "field" and list format

Reply:
DITTO all previous edits to the same table/list text below.
Could we avoid repetition by having just one configuration parameters section that applies to both config methods?